exercises/protein-translation/test/Tests.hs

```diff
@@ -0,0 +1,95 @@
+cases = [ Case { description = "identifies methionine codon"
+               , strand      = "AUG"
+               , expected    = ["Methionine"]
+               }
+        , Case { description = "identifies phenylalanine codon (UUU)"
+               , strand      = "UUU"
+               , expected    = ["Phenylalanine"]
+               }
+        , Case { description = "identifies phenylalanine codon (UUC)"
+               , strand      = "UUC"
+               , expected    = ["Phenylalanine"]
+               }
+        , Case { description = "identifies leucine codon (UUA)"
+               , strand      = "UUA"
+               , expected    = ["Leucine"]
+               }
+        , Case { description = "identifies leucine codon (UUG)"
+               , strand      = "UUG"
+               , expected    = ["Leucine"]
+               }
+        , Case { description = "identifies leucine codon (UUG)"
+               , strand      = "UUG"
+               , expected    = ["Leucine"]
+               }
+        , Case { description = "identifies serine codon (UCU)"
+               , strand      = "UCU"
+               , expected    = ["Serine"]
+               }
+        , Case { description = "identifies serine codon (UCC)"
+               , strand      = "UCC"
+               , expected    = ["Serine"]
+               }
+        , Case { description = "identifies serine codon (UCA)"
+               , strand      = "UCA"
+               , expected    = ["Serine"]
+               }
+        , Case { description = "identifies serine codon (UCG)"
+               , strand      = "UCG"
+               , expected    = ["Serine"]
+               }
+        , Case { description = "identifies tyrosine codon (UAU)"
+               , strand      = "UAU"
+               , expected    = ["Tyrosine"]
+               }
+        , Case { description = "identifies tyrosine codon (UAC)"
+               , strand      = "UAC"
+               , expected    = ["Tyrosine"]
+               }
+        , Case { description = "identifies cysteine codon (UGU)"
+               , strand      = "UGU"
+               , expected    = ["Cysteine"]
+               }
+        , Case { description = "identifies cysteine codon (UGC)"
+               , strand      = "UGC"
+               , expected    = ["Cysteine"]
+               }
+        , Case { description = "identifies tryptophan codon"
+               , strand      = "UGG"
+               , expected    = ["Tryptophan"]
+               }
+        , Case { description = "translate RNA strand into correct protein"
+               , strand      = "AUGUUUUGG"
+               , expected    = ["Methionine", "Phenylalanine", "Tryptophan"]
+               }
+        , Case { description = "stops translation if the STOP codon is present"
+               , strand      = "AUGUUUUAA"
+               , expected    = ["Methionine", "Phenylalanine"]
+               }
+        , Case { description = "stops translation of longest strand"
```

Comment:
I don't understand what "longest" means in this context. Can you clarify? I know that it is the longest strand used in the test cases so far, but I don't understand why that must be specified in the description. To me the important part of this test case is that even if there are codons after STOP, they are not included in the output. Shouldn't that be the description?